symengine/ruby/spec/basic_spec.rb

```diff
@@ -16,56 +16,5 @@
                 end
             end
         end
-
-        describe 'binary operations' do
-            before :each do
-                @a = SymEngine::Basic.new
-                @b = SymEngine::Basic.new
-            end
-            describe '#+' do
-                context 'with another Basic object as argument' do
-                    xit 'returns a Basic object that is result of
-                        self added to the argument' do
```

Comment:
Right. You can follow the Python tests: https://github.com/sympy/symengine/blob/062a20759808ff4f657e879bcb03818c51a75094/symengine/python/symengine/tests/test_arit.py